src/main/webapp/scripts/serverclient.js

```diff
@@ -40,23 +40,34 @@ class ServerClient {
      * @private {URLSearchParams}
      */
     this.urlParams_ = urlParams;
+
+    this.startSessionCache_();
   }
 
   /** 
    * This method starts to cache.
    * @private
    */
   startSessionCache_() {
-    throw new Error('Unimplemented');
+    this.sessionCache_.start();
   }
 
   /**
    * This method changes the controller of the current session to the 
-   * Attendee pased in.
+   * Attendee passed in.
    * @param {string} name 
    */
   passController(name) {
```

Comment:
Maybe rename this to changeController or something. Rename the parameter from "name" to "newControllerName" or something more descriptive.

Reply:
And change the servlet name as well